### DISCUSSION TITLE
No suitable API installation found.

### DISCUSSION THREAD

**Discussion by juanchehin** [ionic-team/capacitor#4863] [Q&A]
Hi, when I run 'npx cap run android' I get the following error. Any solution?
ERR_UNSUITABLE_API_INSTALLATION: No suitable API installation found.

![cap](https://user-images.githubusercontent.com/40132117/126874024-cf0185ac-28ad-4b92-bd86-4cf09ebdf8b6.png)

**Comment by pwespi** [comment]
There is a link in your error message. In the wiki it says:

> ERR_UNSUITABLE_API_INSTALLATION
You are using an Android SDK that is not supported by native-run. Please, let us know by creating a new issue https://github.com/ionic-team/native-run/issues. As workaround you can downgrade your installed Android SDK. Latest supported version is SDK 30.

https://github.com/ionic-team/native-run/wiki/Android-Errors#err_unsuitable_api_installation
